### ISSUE TITLE
Region is not passed in during deployment

### ISSUE THREAD

**Issue by herschel666** [architect/deploy#13]
Hi.

The `sam package`- & the `sam deploy`-commands have a `--region`-flag, which is currently not passed in, if I get this right. However adding it to the spawning of both commands inside of `@architect/deploy` in my node_modules folder fixed the deployment for me, while before it constantly errored with…

```
TypeError: Cannot destructure property `url` of 'undefined' or 'null'.
    at done (server/node_modules/@architect/deploy/src/sam/02-after/00-get-app-apex/index.js:12:16)
    at done (server/node_modules/@architect/deploy/src/sam/02-after/00-get-app-apex/reads.js:25:14)
    at end (server/node_modules/run-parallel/index.js:18:15)
    at done (server/node_modules/run-parallel/index.js:22:10)
    at each (server/node_modules/run-parallel/index.js:28:7)
    at Response.<anonymous> (server/node_modules/run-parallel/index.js:38:43)
    at Request.<anonymous> (server/node_modules/aws-sdk/lib/request.js:366:18)
    at Request.callListeners (server/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (server/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (server/node_modules/aws-sdk/lib/request.js:714:14)
```

Plus: by adding a `console.log` to `aws-sdkk@2.503.0/lib/request.js:364` I got this additional error message…

```
 ValidationError: Stack with id *******Staging does not exist
```

Cheers.

**Comment by ryanblock** [comment]
I've got a fix for this, should be out in the next version, please stand by!

**Comment by brianleroux** [comment]
fixed in `+ @architect/deploy@1.0.19` / updating cli shortly!
